### PR TITLE
added the instruction of install Qt4 for macOS Sierra 10.12.4

### PR DESCRIPTION
**Pull request by yinghe616**
Added instruction into the manual on installing the qt4 on a Mac machine with  macOS Sierra 10.12.4

### CODE REVIEW

File: doc/manual/manual.tex [hunk 1]
```diff
@@ -4192,6 +4192,26 @@ \subsubsection{Installing parameter-GUI}
...
+brew tap cartr/qt4
+brew tap-pin cartr/qt4
+brew install qt@4
+\end{verbatim}
+or install it through Mac Ports (\url{https://www.macports.org/})
+\begin{verbatim}
+sudo port install qt4-mac
+\end{verbatim}
+Then you can follow the Linux user instructions provided previously to download and install dealii parameter-GUI. 
+Before running \texttt{cmake .}, you many need to either pass the path of \texttt{qt4} and specify the value of variable \texttt{QT\_LIBRARIES} to the directory that contains the libraries of \texttt{qt4}
```
Comment (gassmoeller) [inline doc/manual/manual.tex]: you may

File: doc/manual/manual.tex [hunk 2]
```diff
@@ -4192,6 +4192,26 @@ \subsubsection{Installing parameter-GUI}
 to the directory that contains the parameter-GUI executable (optional). 
 This will allow \aspect{} to automatically enable the GUI during configuration.
 \end{enumerate}
+On a mac machine with recent macOS Sierra 10.12.4, Qt development libraries of version 4.x.x at the libraries' official website  \url{https://www.qt.io/download-open-source/} may fail to install.  Alternatively, you can install \texttt{qt4} through Homebrew (also see instruction here \url{https://github.com/cartr/homebrew-qt4})
```
Comment (gassmoeller) [inline doc/manual/manual.tex]: double space before the \url

Comment (gassmoeller) [inline doc/manual/manual.tex]: please put this into a separate \paragraph like `Installing on macOS`

**Comment by yinghe616** [comment]
Comments addressed.

**Pull request merged**